packages/scrum-client/src/content/ParticipateSession/ParticipateSession.js

```diff
@@ -1,30 +1,40 @@
 /* eslint-disable no-console */
 import React, { useEffect, useState } from 'react';
-import { Container } from 'react-bootstrap';
+import {
+  Container,
+  Row,
+  Col,
+  Button,
+  Spinner,
+  Form,
+  ButtonToolbar,
+  ButtonGroup,
+} from 'react-bootstrap';
 import { useParams } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
 import { END_POINTS } from 'scrum-common';
 import TeamList from '../../components/TeamList';
-import SocketEvents from '../../hooks/SocketEvents';
-import { API_BASE_URL } from '../../constants/apiConstants';
-import { useAuthState } from '../../context';
+import useSocket from '../../hooks/useSocket';
+import { API_CONSTANTS } from '../../constants';
 import './ParticipateSession.css';
 
-const getPlanningSession = id => {
+const getPlanningSession = (roomId, setSessionInformation) => {
   const requestOptions = {
     method: 'GET',
     headers: { 'Content-Type': 'application/json' },
   };
 
   try {
-    fetch(`${API_BASE_URL}${END_POINTS.API}${END_POINTS.PLANNING_SESSION}/${id}`, requestOptions)
+    fetch(
+      `${API_CONSTANTS.API_BASE_URL}${END_POINTS.API}${END_POINTS.PLANNING_SESSION}/${roomId}`,
+      requestOptions
+    )
       .then(response => response.json())
       .then(data => {
         if (data.success) {
-          // eslint-disable-next-line
-          console.log(data);
+          setSessionInformation(data.sessionInformation);
         } else {
           console.log('error');
-          // setApiResponse(data.message);
         }
       });
```

Comment:
Chavito, in this cases does it make sense to have the catch after the then? Instead of the try catch block? Not to change it now, just wondering.

Reply:
It depend of how the backend is gonna reply , 
If it is going to reply as http errors, business issues, then  you  are right. 
If business issues are responded as a successful http request but the response of the message change  , then it is needed. 

So the answer at this moment I do not know now ... we need to have a pattern in all our backend apis

Reply:
Makes sense